hub/src/Website/SolarSENSE/models.py

```diff
@@ -27,3 +27,11 @@ def __str__(self):
     name = models.CharField(max_length=255)
     content = models.FileField()
     tags = TaggableManager(blank=True)
+
+
+class Collections(models.Model):
+    date_time = models.DateTimeField(auto_now_add=True)
```

Comment:
You should add a default value for each sensor field. From what I remember the API methods set one field each. As such you wouldn't be able to instantiate the Collections object with all of its fields at once.